catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/ResultDAGConverter.java

```diff
@@ -84,6 +84,53 @@
   private static final Pattern ATTRIBUTE_PATTERN =
       Pattern.compile("([a-zA-Z0-9_:]+):([a-zA-Z0-9_]+).([a-zA-Z0-9]+)");
 
+  private static Map<String, String> typeConversionMap;
+
+  private static boolean forceHttp;
+
+  static {
+    typeConversionMap = new HashMap<>();
+    typeConversionMap.put("Collection", "COLLECTION/EXPLOITATION PLAN");
+    // Text can be any of DOCUMENT, MESSAGE, CBRN, and others - will be checked later
+    typeConversionMap.put("Text", "DOCUMENT");
+    // typeConversionMap.put("Text", "GEOGRAPHIC AREA OF INTEREST");
+    // typeConversionMap.put("", "GEOSPATIAL VECTOR");
+    // typeConversionMap.put( "", "GMTI");
+    typeConversionMap.put("Image", "IMAGERY");
+    // typeConversionMap.put("", "INTELLIGENCE REQUIREMENT");
+    // typeConversionMap.put("Text", "MESSAGE");
```

Comment:
why are these commented out?

Reply:
See https://github.com/codice/alliance/pull/738#discussion_r316820347. This mapping will be completed later.